targets/TARGET_STM/pwmout_api.c

```diff
@@ -39,6 +39,43 @@
 
 static TIM_HandleTypeDef TimHandle;
 
+/* Convert STM32 Cube HAL channel to LL channel */
+uint32_t TIM_ChannelConvert_HAL2LL(uint32_t channel, pwmout_t *obj)
+{
+#if !defined(PWMOUT_INVERTED_NOT_SUPPORTED)
+  if (obj->inverted) {
```

Comment:
Minor comment - TAB size in mbed-os is 4.

Reply:
Done